integration/tests/sign-out-smoke.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from '@playwright/test';
+
+import { appConfigs } from '../presets';
+import type { FakeUser } from '../testUtils';
+import { createTestUtils, testAgainstRunningApps } from '../testUtils';
+
+testAgainstRunningApps({ withEnv: [appConfigs.envs.withEmailCodes] })('sign out smoke test @generic', ({ app }) => {
+  test.describe.configure({ mode: 'serial' });
+
+  let fakeUser: FakeUser;
+
+  test.beforeAll(async () => {
+    const u = createTestUtils({ app });
+    fakeUser = u.services.users.createFakeUser();
+    await u.services.users.createBapiUser(fakeUser);
+  });
+
+  test.afterAll(async () => {
+    await fakeUser.deleteIfExists();
+    await app.teardown();
+  });
+
+  test('sign out throught all open tabs at once', async ({ page, context }) => {
+    const mainTab = createTestUtils({ app, page, context });
+    await mainTab.po.signIn.goTo();
+    await mainTab.po.signIn.setIdentifier(fakeUser.email);
+    await mainTab.po.signIn.continue();
+    await mainTab.po.signIn.setPassword(fakeUser.password);
+    await mainTab.po.signIn.continue();
+    await mainTab.po.expect.toBeSignedIn();
+
+    await mainTab.tabs.runInNewTab(async m => {
+      await m.page.goToStart();
+
+      await m.page.waitForClerkJsLoaded();
+
+      await m.po.expect.toBeSignedIn();
+
+      await m.page.evaluate(async () => {
+        await window.Clerk.signOut();
+      });
+    });
+
+    expect(await mainTab.page.evaluate('!window.Clerk.user')).toBe(false);
```

Comment:
🙃 let's add a `await m.po.expect.toBeSignedOut();` test helper and use it here.

Reply:
The helper exists but there is a problem with it, the `page.waitFucntion` that is underneath `mainTab.po.expect.toBeSignedOut()` helper is getting a timeout at this point, I'm not exactly sure why is that, but it's probably has something to do with with the new tab we are opening.

Reply:
let's add a comment about why we cannot use the helper and fix this later.